### ISSUE TITLE
A11y: Category toggles have no label

### ISSUE THREAD

**Issue by nlehmkuhl** [eibiflo/cf_cookiemanager#56]
The template defines a <span class="t-lb"> for this, however the javascript consent.js does not populate it.
Will provide a pull request for this.
